README.md

```diff
@@ -1,24 +1,54 @@
 # Lazyman.bundle
 Lazyman Plex Channel to access Lazyman streams along with recaps and extended highlights.
 
-## Streaming and transcoding
+
+## Streaming and Transcoding
 As far as I gather, the Plex Media Server where the plug-in is installed need to be powerful enough to transcode the HLS (HTTP Live Streaming) streams.
 It might work with a transcoding client such as Plex Media Player or OpenPHT on a computer, not verified.
 Installing the plug-in on a weaker NAS and using a limited client will not work.
 
-### Recaps and highlights
-Recaps and highlights are not part of the Lazyman solution but is offered in the same NHL API and are added for convenience. These are MP4 files and not streams so they should generally work better on most devices. They will also work (or not work) without `/etc/hosts` changes.
+### Recaps and Highlights
+Recaps and highlights are not part of the Lazyman solution but is offered in the same NHL API and are added for convenience. These are MP4 files and not streams so they should generally work better on most devices. They will also work (or not work) without changes to your `hosts` file.
+
+
+## Prerequisites
+### Modify hosts file
+ * Modify your `hosts` file to work with Lazyman on the machine running Plex Media Server:
+ 1. From a command prompt or terminal window, type `ping nhl.freegamez.gq` and note the IP address supplied in the output.
+ 2. Locate the hosts file for your OS.
+    * *The location of this file can vary depending on OS so if you don't know it's best to google for something like "**hosts file location [your OS/NAS here]**".*
```

Comment:
Why don't you rather (also) link to a guide for different OSes, like https://www.siteground.com/kb/how_to_use_the_hosts_file/ or something better you can find on Google? :)

Reply:
Agreed. Was taking the "teach a man to fish" approach, but an example of a good article that covers  many of the popular locations would be helpful. I'll get this updated with your link and do a secondary round of searching (even though yours is probably more than adequate).

Reply:
Added the link you provided and cleaned up the list a bit. I will keep my eyes open for anything else that might look like it can help people modify hosts files, but like I said before, your link should cover most (if not all) of the systems that Plex runs on.